backend/src/achievements-log/entities/achievements-log.entity.ts

```diff
@@ -19,13 +19,13 @@ export class AchievementsLog {
   createdAt: Date;
 
   @Exclude()
-  @ManyToOne(() => User)
+  @ManyToOne(() => User, { onDelete: 'CASCADE' })
```

Comment:
This is a good change, but it seems out of scope. Keeping our PRs short guarantees a faster review time.